packages/xo-server/src/api/pci.mjs

```diff
@@ -0,0 +1,43 @@
+import { asyncEach } from '@vates/async-each'
+import { defer } from 'golike-defer'
+
+export const hide = defer(async function ($defer, { pcis, hide }) {
```

Comment:
should the method be called isEnable and set({enable: true}) like in pusb  ?

Reply:
IMO, `isEnable` can be confusing.
I guess at `xo-server` level it means PCI is enabled for passthrough, but at hypervisor level `is_dom0_access_enabled` means PCI is not ready for passthrough.

Also, `Hidden` refers to the [XCP-ng doc](https://docs.xcp-ng.org/compute/#2-tell-xcp-ng-not-to-use-this-device-id-for-dom0) (`xen-pciback.hide`)